pkg/build/buildkit/connector/portforward.go

```diff
@@ -43,6 +45,14 @@ var waitReasonMessages = map[string]string{
 	"PODS_SCALING":      "BuildKit pods are starting up",
 }
 
+const (
+	defaultMaxWaitTimePortForward  = 10 * time.Minute
+	initialPollIntervalPortForward = 1 * time.Second
+	maxPollIntervalPortForward     = 10 * time.Second
+	backoffMultiplierPortForward   = 2.0
+	buildkitPodWaitTimeoutEnvVar   = "OKTETO_BUILDKIT_POD_WAIT_TIMEOUT"
```

Comment:
```suggestion
	buildkitPodWaitTimeoutEnvVar   = "OKTETO_BUILDKIT_QUEUE_WAIT_TIMEOUT"
```

As we are going to give more relevance to the queue (it is going to be even documented), I would be explicit about it in the var name